src/Assignments/ProblemIframe.tsx

```diff
@@ -137,8 +139,8 @@ export const ProblemIframe: React.FC<ProblemIframeProps> = ({
     }
 
     function insertListener(problemForm: HTMLFormElement) {
-        problemForm.addEventListener('submit', _.debounce((event: { preventDefault: () => void; }) => {
-            event.preventDefault();
+        problemForm.addEventListener('submit', (e) => { e.preventDefault();});
+        problemForm.addEventListener('submit', _.debounce(() => {
```

Comment:
I'm not sure about the behavior of `preventDefault` across multiple listeners. I think I'd prefer that we combine the two calls into one as you had before, with the debounce logic _after_ the `preventDefault()`.

Reply:
updated